app/src/components/market/sections/market_create/steps/items/funding_and_fee_step.tsx

```diff
@@ -207,6 +247,20 @@ const FundingAndFeeStep: React.FC<Props> = (props: Props) => {
     setAllowanceFinished(false)
   }
 
+  const toggleCustomFee = () => {
+    if (customFee) resetTradingFee()
+    if (!customFee) setFee(undefined)
+    setCustomFee(!customFee)
+  }
+
+  useEffect(() => {
+    const setMaxFeeWarning = () => {
+      spread > MAX_MARKET_FEE ? setExceedsMaxFee(true) : setExceedsMaxFee(false)
+    }
+
+    setMaxFeeWarning()
+  })
```

Comment:
This should be giving a warning about the dependency list of this effect, there is no need of a function here.
```
useEffect(() => {
  setExceedsMaxFee(spread > MAX_MARKET_FEE)
}, [spread])
```